### PR TITLE
update zhCN.

### PR DESCRIPTION
**Pull request by adavak**
update zhCN.
cleanup.

### CODE REVIEW

File: Core/Locales/common.zhCN.lua [hunk 1]
```diff
@@ -8,55 +8,55 @@ L.adds = "增援"
...
+L.onboss = "首领%s！"
+L.other = "%s：%s"
+L.over = "%s结束！"
 L.phase = "阶段%d"
-L.removed = ">%s<移除"
-L.removed_from = "%2$s 已移除 >%1$s<"
---L.removed_by = "%s removed by %s" -- spell removed by player
+L.removed = "%s移除"
+L.removed_from = "%2$s已移除%1$s"
+removed_by = "%s被%s移除" -- spell removed by player
```
Comment (funkydude) [inline Core/Locales/common.zhCN.lua]: You cut out the `L.`

**Comment by funkydude** [comment]
I'm happy to see you are removing the `<>` but, why now?

**Comment by adavak** [comment]
same with other locale, and simple is best.

**Comment by adavak** [comment]
done.

**Pull request merged**